app/client/apps/goat/components/map/panels/MapStyle.tsx

```diff
@@ -1,9 +1,235 @@
 import Container from "@/components/map/panels/Container";
+import { IconButton, Text } from "@p4b/ui/components/theme";
+import {
+  Button,
+  CardContent,
+  CardMedia,
+  Divider,
+  FormControlLabel,
+  Radio,
+  RadioGroup,
+  Typography,
+} from "@mui/material";
+import { makeStyles } from "@/lib/theme";
+import { useDispatch, useSelector } from "react-redux";
+import type { IStore } from "@/types/store";
+import { setTabValue } from "@/lib/store/styling/slice";
+import { Card } from "@p4b/ui/components/Surfaces";
+import StarIcon from "@mui/icons-material/Star";
+import Tabs from "@mui/material/Tabs";
+import Tab from "@mui/material/Tab";
+import { Checkbox } from "@p4b/ui/components/Checkbox";
+import BasicAccordion from "@p4b/ui/components/BasicAccordion";
+import { MultipleSelect } from "@p4b/ui/components/Inputs/MultipleSelect";
+import ArrowDropDownIcon from "@mui/icons-material/ArrowDropDown";
+import Box from "@p4b/ui/components/Box";
+import type { MapSidebarItem } from "@/components/map/Sidebar";
 
-import { Text } from "@p4b/ui/components/theme";
+interface MapStyleProps {
+  setActiveRight: (item: MapSidebarItem | undefined) => void;
+}
+const MapStylePanel = ({ setActiveRight }: MapStyleProps) => {
+  const { tabValue } = useSelector((state: IStore) => state.styling);
 
-const MapStylePanel = () => {
-  return <Container header={<Text typo="page heading">Style</Text>} body={<Text typo="body 1">Body</Text>} />;
+  const dispatch = useDispatch();
+
+  const layerTypes = [
+    {
+      label: "@column_label",
+      value: "@column_label",
+    },
+    {
+      label: "@column_label1",
+      value: "@column_label1",
+    },
+  ];
+
+  const layerAccordionInfo = [
+    {
+      id: 1,
+      title: "Marker",
+    },
+    {
+      id: 2,
+      title: "Color",
+    },
+    {
+      id: 3,
+      title: "Stroke",
+    },
+    {
+      id: 4,
+      title: "Size",
+    },
+  ];
+
+  const { classes } = useStyles();
+
+  const handleChange = (_event: React.SyntheticEvent, newValue: number) => {
+    dispatch(setTabValue(newValue));
+  };
+
+  return (
+    <Container
+      header={
+        <Box className={classes.contentHeading}>
+          <IconButton iconId="chevronRight" iconVariant="focus" onClick={() => setActiveRight(undefined)} />
```

Comment:
It would be better to use the font awesome icons. IconButton still uses the material icons, so this might result in inconsistency.